### DISCUSSION TITLE
Versions compatibility for ConfigurationManager and DataspaceConnector

### DISCUSSION THREAD

**Discussion by mikhail-gordienko** [International-Data-Spaces-Association/DataspaceConnector#70] [Q&A] [answered]
Hello!

What versions of ConfigurationManager and ConfigurationManager-UI are compatible with DataspaceConnector 4.3.0?

Documentation in the ConfigurationManager-UI repository states the following:

_IDS Configurationmanager and the UI versioning skipped major version 1, 2 and 3 to match versioning of IDS DataSpaceConnector. This way the major version of DSC, CM and UI indicates compatibility._

But there is no v4 tag in the ConfigurationManager repository. I wonder if it is possible to perform the same demo as [the following](https://www.youtube.com/watch?v=zk0N6u5ewOA) but with v4 for all three components.

**Comment by ghost** [comment, marked as answer]
Hello and thank you for the discussion-post!

For both ConfigManager and ConfigManager-GUI, the compatible versions are 7.0.0.

(the docu in the GUI-Project seems to be outdated at that point, sorry for that!)
